db/migrate/20160329231848_create_thredded.rb

```diff
@@ -162,16 +162,18 @@ def change
 
     create_table :thredded_user_preferences do |t|
       t.references :user, null: false
-      t.boolean :notify_on_mention, default: true, null: false
+      t.boolean :auto_follow_topics, default: true, null: false
       t.boolean :notify_on_message, default: true, null: false
+      t.boolean :followed_topic_emails, default: true, null: false
       t.timestamps null: false
       t.index [:user_id], name: :index_thredded_user_preferences_on_user_id
     end
 
     create_table :thredded_user_messageboard_preferences do |t|
       t.references :user, null: false
       t.references :messageboard, null: false
-      t.boolean :notify_on_mention, default: true, null: false
+      t.boolean :auto_follow_topics, default: true, null: false
+      t.boolean :followed_topic_emails, default: true, null: false
```

Comment:
Correct me if I'm wrong - but this is the migration for the very first creation of the thredded tables, correct? So it's not a problem that we're updating this particular migration?

I'm mostly talking out loud here about it but figured I would confirm :)